### ISSUE TITLE
mapathon

### ISSUE THREAD

**Issue by GretaTimaite** [udsleeds/openinfra#109]
Let's organise a mapathon in Leeds focused on accessible active travel infrastructure?

**Comment by GretaTimaite** [comment]
youthmappers could help us with organisation. Alas, there's no chapter at Leeds Uni.

**Comment by hulsiejames** [comment]
Missing maps could be helpful (from our teams chat Greta), though this is focused on hosting mapathons to map potential crisis areas to aid in crisis response it seems.

![Screenshot from 2022-08-21 14-47-07](https://user-images.githubusercontent.com/58815827/185794000-040f9f69-5713-49ee-a91d-714c4e212ac0.png)

From: https://www.missingmaps.org/host/#helper-map-contents

**Comment by GretaTimaite** [comment]
This is active travel crisis @hulsiejames!

**Comment by GretaTimaite** [comment]
My initial thoughts about the whole thing are the following:
1. I do not think we need more than 20 mappers (including organisers) for the mapathon as it could be hard to manage. 
2. Timings: I reckon it will either have to be 30 Sept (or MAYBE 3 Oct) or end of October. I'm starting my physio in October, so I might be unable to simply jump on plane to travel to the UK for the day.
3. Food: we should provide some refreshments + snacks for the mappers.
4. Venue: We might not need a computer cluster as we will be "going out there" to do the mapping with the phones. We will need, however, a room for training (install an app, how to use/map, why important, etc)

**Comment by hulsiejames** [comment]
> 1. What is the anticipated resource needed

_Finance only or finance and staff support time? Can you estimate a figure you might potentially need_ 
- I would probably say only finance to cover the cost of travel and providing snacks/lunch etc. unless we need to request staff time for food?

> 2. Venue

_computer cluster 11.06 next to LIDA may be available for this depending on the date, and has capacity for 50 persons_
- Provided this cluster is available I can see this fitting our needs. Access to computers so can cover editing using the OSM editor and any other relevant learning materials. Also, I think a capacity of 50 is more than enough for what we need (approx. 20?)

> 3. Timings

_it seems wise to go for a date in October as this avoids the intensive teaching roster in 11.06 with the new term start in September_
- I think this sounds appropriate too - I will still in Leeds for at least the first half of October for sure, potentially a little longer and as mentioned this avoids the busy teaching roster, and the bust first week of the new LIDA data scientists

**Comment by GretaTimaite** [comment]
Today we're meeting Gregory and Kim, so here's a list of things that we might want to discuss:

1. TIME: 
     - date and time for the event
     - duration
2. SCALE: 
     - how many people do we expect to attend? 
     - what's the maximum number of participants we prepared to manage?
3. AUDIENCE:
     - who's our target audience? 
          - LIDA
          -  School of Geography
          -  Institute for Transport Studies
      - engaging those outside of academic community? Social media?
4. LOCATION:
     - venue for gathering and training
     - mapping location(s)
5. EVENT:
     - introduction (rationale behind mapathon)
     - training (what participants need to know to map)
     - field work (mapping): 
           - how will it be structured? group or individual mapping, splitting based on...
     - end of the day 
          - food + beverages? 

I'm sure more questions will arise as we start discussing these things but it's a starting point? Feel free to add any other points that pop in your mind.

=====

Also, I think it might be interesting to measure the impact of this mapathon through added data. I can come up with a couple of simple ways of doing it.

First approach:
- download OSM data of a mapped region a day before an event, focus on active travel tags
- download OSM data of a mapped region a day after an event
- compare them 

Second approach:
- use ohosme

**Comment by hulsiejames** [comment]
These questions look really good and on point. I can't think of anything I would add for now, but will have a muse over lunch. 

I think Gregory should be able to help with a lot of the specifics of organising a mapathon (max participants, timings of event etc.)

Looking forward to the call.

**Comment by hulsiejames** [comment]
Incredibly rough meeting notes on [HackMD](https://hackmd.io/FO44m8QXTta6We8s_gVdPw).
I am writing these up in neat and will post here...

**Comment by GretaTimaite** [comment]
Great, many thanks James for doing this! 

After the meeting I thought we could have asked to record it 😅

**Comment by hulsiejames** [comment]
**TIME**
> date and time for the event
- End of October early November (dependant on LIDA room availablilty). Gregory mentioned that you don't really need more than 4 weeks notice unless you're looking to host a large event (nation wide reach). Sautdays & Sundays obviously most desireable with work/academic commitments during the week, but if required Mondays and Fridays work best during the week. 
> duration
- Gregory mentioned that when doing detailed mapping, it becomes quite intensive after an hour of mapping so shorter is sweeter, but suggested two mapping sessions with tea/coffe/snacks inbetween, say 45-60mins mapping, 30-60 min break, another 45-60mins mapping. 
- Max 2 hours for into talk, what is OSM and why is it important, how to map features, what tags are actually useful for active travel/accesibility (this can probably be done in less than 2 hrs). 
- Finally once mapping is over, meet up for final drink/snacks, and a discussion. Back in the day this was a really useful part of the mapathon as people could discuss mapping ideas and features they discovered whilst mapping "I found this intresting bicycle barrier, how should we be mapping these" etc. Gergory mentioned that **this part was always really useful**. 
- All in all, perhaps max 6 hours give or take(including an intro talk, tiny break, first mapping session, break, second mapping session, final discussion and snacks)

**SCALE**
>how many people do we expect to attend?
>what's the maximum number of participants we prepared to manage?

- Really it seems that we want, at max, a classroom size of people. Gregory suggested 15-20 as a good max number for a first mapathon, splitting into teams of 2 or 3 to go out mapping

**AUDIENCE**
> who's our target audience?
- Gregory mentioned that it is helpful (but not crucial) to have a group of mappers who feel they can get something from the mapping event... i.e a local walking group would likely be a good target when looking to map local walking routes, similarly a local cycling group is useful for mapping cycling infrastructure, acadmeics who may have intrests in analysis of specific types of OSM data, etc.
> LIDA, School of Geography, ITS
- All seem good suggestions to myself

**LOCATION** 
> venue for gathering and training
- Believe LIDA can facilitate - follow up with email. 
> mapping location(s)
- Gregory mentioned the concept of cake slices. Once you have identified an area you would like to be mapped (I asusme centred at the host location), draw a circle around this area. Then, split the circle into numbered segments. When mappers come to the mapathon, allow them to select a segment or two in the teams of two or three, this is then the segments they will walk in the mapathon. 
- Building on this, we could plot the density of tags describing active travel infrastructure in Leeds as a heatmap, identify cells of the heatmaps with the lowest densities of active travel tags and use these as our segemtns?  

**EVENT**
From the above, the general structure could be something like: 
- Introduction (basic concept, motivation, how to map, select segments. 1.25-2 Hrs)
- 5-10 min break
- First mapping session (45-60 mins)
- break (30-45) [this could be lunchtime?]
- Seoncd mapping session (45-60 mins)
- Post mapping discussions/snacks/drinks (30-60-?mins)


_________________________________________________________________

**General comments from meeting**

- It is a good idea to have a mapathon focused on active travel / accessible active travel (in general a specific focus) as Leeds (in general) is already mapped quite well. Gregory liked the idea of using Streetcomplete and IOS go (I think is the name) as facilitates mapping on the go, rather than notepad & pen --> computer editor approach. 

- Gregory mentioned that mostly individual or small teams (2/3) of mappers is a better idea, with a more experience mapped joinging a team should all team members be novice. 

- Gregory can do an introduction into OSM, what is OSM and why is it important, how OSM gets used by people and why it is useful to map the world for OSM, and how to map for OSM. Obviously we can split some of this up so not such a large workload

- It is a good idea to have someone in the host location at all times. If mappers are getting bored before an hour session is up, or they have already mapped their segments then they can come back to the lcoation, get an early drink and have a chat with one of the hosts, or even ask for a new, nearer location.

**Comment by hulsiejames** [comment]
> After the meeting I thought we could have asked to record it

I wish we had :grinning:

**Comment by Robinlovelace** [comment]
Sounds like a productive meeting! Sorry for missing it, look forward to helping to make it happen, exactly what is needed.

**Comment by GretaTimaite** [comment]
@hulsiejames and I had a quick chat yesterday with Kim. Here are a few key takeaways:

- Organising an event on a weekend would require much more effort and time as additional safety and security approvals would be needed
     - thus, best bet is Monday or Friday
- we capped the number of participants (excluding us) at 20
     - thus, computer cluster at LIDA should do as a venue
     - we will need to set up visitor ID (is it how it's called) to access computers, if needed. This includes James and me :/ 
- providing food and drinks will not be a problem
- Kim offered herself to set up an EventBrite event, but we need to send all the details, such as
     - date
     - description of an event (including preliminary schedule, mention food, etc.)
     - organisers
     - to which departments it should be forwarded as an email

I think the **most important consideration is date**. Looking at the calendar I think the following days are apt:

- Fri 28 Oct
- Mon 31 Oct (Halloween?)
- Fri 4 Nov
- Mon 7 Nov
- Fri 11 Nov

**Any days that you for sure know you cannot make** @hulsiejames and @Robinlovelace?  Or any other suggestions? This should be a half-day event in the morning/afternoon, fyi. If all of these days look good for you then I will send the dates to Gregory to inquire if there's the one he prefers most?

**Comment by Robinlovelace** [comment]
Many thanks for instigating this Greta, any of those look good to me!

**Comment by hulsiejames** [comment]
Hi Greta, Friday and Monday (28th / 31st) work best for me as my tenancy runs out onf Friday 4th November.

**Comment by GretaTimaite** [comment]
I'm not sure how much Halloween on 31st Oct might be an issue? But I'll send an email to Gregory ASAP to find out which days work best for me and will highlight that these are the days that we have some preference for.

**Comment by hulsiejames** [comment]
>I'm not sure how much Halloween on 31st Oct might be an issue?

Hopefully not too much as we are planning on arranging this within the day.

**Comment by GretaTimaite** [comment]
Just thinking if young people might be more into prepping for the Halloween rather than going to a mapathon, but I don't know what's the Halloween culture in the UK.

**Comment by Robinlovelace** [comment]
Halloween themed mapathon? Could be amazing, thinking 'trick or map'!

**Comment by GretaTimaite** [comment]
I forgot to link my commit to this issue. So to keep it updated, here's a link to the mapathon abstract: https://github.com/udsleeds/openinfra/tree/main/abstracts

**Comment by GretaTimaite** [comment]
Just to confirm, @Robinlovelace and @hulsiejames you're on board with the mapathon description? I really haven't had time this week to follow up with you or Kim but will send an email to her by Monday morning to confirm the date (2022-10-31), so she can get the required approvals in time.

**Comment by Robinlovelace** [comment]
I'm onboard, please send a link to the latest version, should know where that is :see_no_evil:

**Comment by GretaTimaite** [comment]
Sorry, I forgot to paste it. See .Rmd here: https://github.com/udsleeds/openinfra/blob/main/abstracts/mapathon.Rmd

**Comment by GretaTimaite** [comment]
Feel free to edit it, so I have the final version.

**Comment by hulsiejames** [comment]
> Gregory mentioned the concept of cake slices. Once you have identified an area you would like to be mapped (I asusme centred at the host location), draw a circle around this area. Then, split the circle into numbered segments. When mappers come to the mapathon, allow them to select a segment or two in the teams of two or three, this is then the segments they will walk in the mapathon.

On this - I want to create a heatmap around LCC based on the density of active travel OSM features (kerb tags, tactile paving tags, etc - additioanal thoughts welcome). 

Based on this, we can then create the 'slices' / segments to for mappers to visit. 

Thoughts and suggestions are welcome!

**Comment by GretaTimaite** [comment]
> On this - I want to create a heatmap around LCC based on the density of active travel OSM features (kerb tags, tactile paving tags, etc - additioanal thoughts welcome). 
> 
> 
> 
> Based on this, we can then create the 'slices' / segments to for mappers to visit. 
> 
> 
> 
> Thoughts and suggestions are welcome! 

We could do this but I'm not sure it's really necessary? And how would you measure density that is not a rather crude approximation?

I think the easiest thing to do is split LCC into equal parts, or maybe by street so we can later use ohsmore (is it what the osm history tool is called?)  to explore the difference a mapathon made. Perhaps we can send results to the participants too (I think Gregory suggested this). We can mention this in the email as an incentive. 

But if you want and have time to play with the heatmap idea -- go ahead. Even if it fails, it's good to have an empirical evidence of why. 

An idea while typing this: maybe we could print out maps so participants can highlight what was covered.

**Comment by hulsiejames** [comment]
>And how would you measure density that is not a rather crude approximation?

I suppose it would be an approximation to be honest, my thoughts were if we split LCC into equal parts using a grid and count accessible active travel features within each grid to make the heatmap, as done here but with collision data: https://urbandatapalette.com/post/2021-08-tessellation-sf/

I think this might be mroe of a fun idea to play around with though rather than target where needs mapping. Just because an area has small amount of mapped accessible active travel infrastructure features, it does not necessarily mean this area "has not been mapped" - there may just be lacking infrastructure there.

As such, the best approach may just be sending people out to specific areas so that we are sure these areas have been covered. Thus, any areas with little accessible active travel features will be due to a lack of features, and not "this area just hasn't been mapped yet"

________________________________

> An idea while typing this: maybe we could print out maps so participants can highlight what was covered.

I think this is a very good idea, as allows us to state the above confidently (_Thus, any areas with little accessible active travel features will be due to a lack of features, **and not "this area just hasn't been mapped yet"**_ <-- As we know which areas have been mapped!)

Another way we could do the above is by looking at user ID's and changesets to see where people have mapped. 

User ID's is a bit tricky I believe - these are not included in data providers extracts (Geofabrik, bbbike, I beieve) but are obtainable from some route. (I remember hearing somewhere, maybe SOTM, that if you query the OSM database when logged in through your own account, one does get access to user ID's)

__________________________________

> and count accessible active travel features within each grid  (My first paragraph)
 
Robin made a good point on this after the ITS hackathon yesterday. 
We should definately try and get involvement from a disability / impaired mobility group(s) as to what features are useful to be mapped to allow route planning for accessible active travel routes, otherwise there may be a number of features we overlook / take for granted!

**Comment by hulsiejames** [comment]
> There's [a research group based at Leeds University](https://inclusivepublicspace.leeds.ac.uk/about-the-inclusive-public-space-project/) with whom I met in January to discuss exactly this. We didn't end up following much after, alas. I can connect you and @Robinlovelace with them. Vikki attended that meeting, so maybe Robin remembers them too. I could try emailing them about the mapathon but it's a bit short notice...

Yes I think this is an excellent Idea. Even if nobody is available to participate / help in the mapathon, consulation on the sort of street features that lead to exclusionary infrastructure is important imo! 

>I think Gregory also suggested asking participants to create accounts, so we can validate tags. I think this is a sensible approach even if takes extra work for us. Perhaps we could create a second event focused on validation :D

Yes I like this - be it creating an account on the day or providing us with the user tag of an existing OSM mapper.

**Comment by Robinlovelace** [comment]
> There's [a research group based at Leeds University](https://inclusivepublicspace.leeds.ac.uk/about-the-inclusive-public-space-project/) with whom I met in January to discuss exactly this. We didn't end up following much after, alas. I can connect you and @Robinlovelace with them. Vikki attended that meeting, so maybe Robin remembers them too. I could try emailing them about the mapathon but it's a bit short notice...

Good point, @GretaTimaite could you try contacting them and asking if they would be interesting in participating?

**Comment by hulsiejames** [comment]
Good points from e-mail discussions. 

**Ameena Berkowitz** 

>I would suggest you get in touch with a local Leeds disability group, and potentially via the Universities in Leeds. A quick google search under Leeds Disability groups brought up several.

Including [ableize](https://www.ableize.com/disabled-groups-and-clubs-by-county/west-yorkshire/leeds/) - [DREAM](https://dreamleeds.org.uk/index.html) - [LDPO](https://ldpo.co.uk/) (_mentioned below_) - [DIAL LEEDS](https://www.dial-leeds.org.uk/) - [UoL Disability Service](https://students.leeds.ac.uk/info/10710/disability_services) also comes to mind along with the [Inclusive Public Space Project](https://inclusivepublicspace.leeds.ac.uk/about-the-inclusive-public-space-project/)


>You could also potentially engage young people with an interest – you could see if Modeshift has an Active Travel Ambassador school who might be keen to get involved.

[Chris Pryor, Active Travel Ambassador Coordinator](https://modeshift.org.uk/modeshift-news/meet-chris-pryor-active-travel-ambassador-coordinator-south-and-west-yorkshire/) (South & West Yorkshire)

>Could also ask Transport for All to let their members know about this. Obviously this is of specific interest for wheelchair users, but might you think about what criteria would be useful for other disabled people to know about that might be mapped at the same time? I expect T4A could help you with this.

[Transport for All](https://www.transportforall.org.uk/)

**Juan P. Fonseca-Zamora**

Has contacted a friend who works with LDPO ([Leeds Disabled People's Organisation](https://ldpo.co.uk/)), although they are currently **not** in Leeds, they will pass on the information to see if anybody from the organisation can join on the day. 
_________________________________________________________________________________________________

I will look at getting an email sent out to these people/organisations tomorrow for event participation and consulation on accesible features that we should be mapping!
